README.md

```diff
@@ -1 +1,3 @@
-# trustly-android-example
\ No newline at end of file
+# About this demo
+
+This app demo has a propose to demonstrate how to implement and use the Android SDK.
```

Comment:
Please add some basic instructions on how to open the project and get up and running quickly. Can be like 3 bullet points, just enough to get started.